src/main/java/com/facebook/openwifirrm/optimizers/tpc/TPC.java

```diff
@@ -84,25 +83,25 @@ public TPC(
 	}
 
 	/**
-	 * Update the tx power choices based on user and allowed channels from deviceConfig
+	 * Determine the new tx power choices based on user and allowed channels from deviceConfig.
+	 *
 	 * @param band the operational band
-	 * @param serialNumber the device
-	 * @param txPowerChoices the available tx powers of the device
-	 * @return the updated tx powers of the device
+	 * @param serialNumber the AP's serial number
+	 * @param txPowerChoices the AP's available tx powers
+	 * @return the AP's updated tx powers
 	 */
 	protected List<Integer> updateTxPowerChoices(
 		String band,
 		String serialNumber,
 		List<Integer> txPowerChoices
 	) {
-		List<Integer> newTxPowerChoices =
-			new ArrayList<>(txPowerChoices);
+		List<Integer> newTxPowerChoices = new ArrayList<>(txPowerChoices);
 
-		// Update the available tx powers based on user tx powers or allowed tx powers
 		DeviceConfig deviceCfg = deviceConfigs.get(serialNumber);
 		if (deviceCfg == null) {
 			return newTxPowerChoices;
 		}
+
 		if (
```

Comment:
This was a general code style choice where we write "paragraphs" (code blocks with no blank lines in between) preceded by a comment. Prefer to keep the comments because it's faster to skim than code.